packages/processor/processor.js

```diff
@@ -433,6 +444,10 @@ class Processor {
             return;
         }
 
+        const text = typeof textOrRoot === "string" ?
```

Comment:
Do this inside the assignment to `this._files[name]` since nothing else depends on it.